mcs/class/corlib/System.IO/FileSystem.NotSupported.cs

```diff
@@ -0,0 +1,32 @@
+namespace System.IO
+{
+	internal static partial class FileSystem
```

Comment:
Why is it internal?

Reply:
This is an internal helper class, see https://github.com/dotnet/corefx/blob/e0cb81d4df9e6279cff5be3659750f1140d977e0/src/System.IO.FileSystem/src/System/IO/FileSystem.Unix.cs for example